network_importer/adapters/nautobot_api/models.py

```diff
@@ -48,9 +48,9 @@ def get_device_tag_id(self):
             return self.device_tag_id
 
         tag = self.diffsync.nautobot.extras.tags.get(name=f"device={self.name}")
-
         if not tag:
-            tag = self.diffsync.nautobot.extras.tags.create(name=f"device={self.name}", slug=f"device__{self.name}")
+            tag = self.diffsync.nautobot.extras.tags.create(
+                name=f"device={self.name}", slug=''.join(c if c.isalnum() else '_' for c in self.name))
```

Comment:
*might* be better to use `-` rather than `_`, since that's more usual in slugs? Not critical though.

Also, looks like this is removing the `device__` prefix from the tags, not sure if that's intentional?

Reply:
I went with underscore rather than dash because of the slug tip here: https://docs.nautobot.com/projects/core/en/stable/user-guides/custom-fields/?h=slug#grouping

Omitting `device__` was inadvertent, sorry.  I'll fix that.